src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -461,7 +498,7 @@ public static CloseableHttpResponse execute(
                       backoffInMilli, retryTimeoutInMilliseconds - elapsedMilliForTransientIssues);
             }
           } catch (InterruptedException ex1) {
-            logger.debug("Backoff sleep before retrying login got interrupted", false);
+            logger.debug(requestIdStr + "Backoff sleep before retrying login got interrupted", false);
```

Comment:
why don't we have parameters here as {}? is the false parameter missing in the log?

Reply:
`false` parameter is a flag used to mask the secrets, but as we do not have any secrets here, I have changed the message to a proper format.